### DISCUSSION TITLE
Does this library support multiplexing over http2 ?

### DISCUSSION THREAD

**Discussion by vinothvk** [uNetworking/uWebSockets#1644] [Q&A]
Does uWebSockets support http2 multiplexing https://datatracker.ietf.org/doc/html/rfc8441 ?

**Comment by uNetworkingAB** [comment]
Only http1.1 and http3

**Reply by vinothvk** [reply]
Thanks. how about web transport over http3 ?

**Reply by uNetworkingAB** [reply]
Yes whenever it becomes widely used it is a goal to support.
